galleon-feature-pack/wildfly-feature-pack-build.xml

```diff
@@ -0,0 +1,40 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~
+  ~ Copyright 2021 Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<build xmlns="urn:wildfly:feature-pack-build:3.1" producer="org.jboss.resteasy.microprofile:galleon-feature-pack">
+    <transitive>
+        <dependency group-id="org.wildfly" artifact-id="wildfly-ee-galleon-pack" translate-to-fpl="true">
+            <name>org.wildfly:wildfly-ee-galleon-pack</name>
+            <packages inherit="true">
+                <!-- Exclude modules no longer provided by RESTEasy -->
+                <exclude name="org.jboss.resteasy.resteasy-yaml-provider"/>
+                <exclude name="org.jboss.resteasy.resteasy-jettison-provider"/>
+                <exclude name="org.jboss.resteasy.resteasy-spring"/>
+            </packages>
+            <default-configs inherit="false"/>
+        </dependency>
+    </transitive>
+    <dependencies>
+        <dependency group-id="org.jboss.resteasy" artifact-id="galleon-feature-pack" translate-to-fpl="true">
```

Comment:
@jamezp What's the meaning of `translate-to-fpl` here? I tried to search Galleon document but seems didn't find any information.

Reply:
This file generates a feature-pack. The `translate-to-fpl` means it will use a `feature-pack-location`. If you look at the zip file in your target directory you'll see a `feature-pack.xml` file. 

```
<?xml version="1.0" ?>

<feature-pack xmlns="urn:jboss:galleon:feature-pack:2.0" location="org.jboss.resteasy.microprofile:galleon-feature-pack:2.0.0.Alpha1">
    <transitive>
        <dependency location="wildfly-ee@maven(org.jboss.universe:community-universe):current#24.0.1.Final">
            <origin>org.wildfly:wildfly-ee-galleon-pack</origin>
            <default-configs inherit="false"/>
            <packages inherit="true">
                <exclude name="org.jboss.resteasy.resteasy-spring"/>
                <exclude name="org.jboss.resteasy.resteasy-yaml-provider"/>
                <exclude name="org.jboss.resteasy.resteasy-jettison-provider"/>
            </packages>
        </dependency>
    </transitive>
    <dependencies>
        <dependency location="org.jboss.resteasy:galleon-feature-pack:5.0.0.Alpha1">
            <origin>org.jboss.resteasy:galleon-feature-pack</origin>
        </dependency>
    </dependencies>
    <default-packages>
        <package name="modules.all"/>
    </default-packages>
</feature-pack>
```